src/styles/index.css

```diff
@@ -0,0 +1,125 @@
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+body {
+  margin: 0;
+  font-family: Roboto, sans-serif;
+  display: flex;
+}
+
+.card {
+  width: 200px;
+  height: 406px;
+  padding: 16px;
+  border-radius: 5px;
+  color: #060b35;
+  box-sizing: border-box;
+}
+
+.img {
+  width: 160px;
+  height: 134px;
+}
+
+.discribe {
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 18px;
+  margin-bottom: 4px;
+  margin-top: 40px;
+}
+
+.code {
+  font-size: 10px;
+  line-height: 14px;
+  color: #616070;
+  margin: 0;
+  margin-bottom: 16px;
+}
+
+.rating {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  text-align: center;
+}
+
+.rating .stars {
+  display: flex;
+  width: 100%;
+  max-width: 250px;
+  justify-content: flex-start;
+}
+
+.rating .stars__star {
+  background-image: url(../images/star.svg);
+  width: 16px;
+  height: 16px;
+  background-size: cover;
+  margin-right: 4px;
+}
+
+.rating .stars--4 .stars__star:nth-child(-n + 4) {
+  background-image: url(../images/star-active.svg);
+}
+
+.rating .reviews {
+  color: #060b35;
+  font-size: 10px;
+  line-height: 14px;
+  width: 400px;
+  margin: 0;
+  text-align: right;
+}
+
+.price {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  line-height: 18px;
+  height: 18px;
+  width: 100%;
+  margin-bottom: 16px;
+  margin-top: 24px;
+}
+
+.price .price {
+  color: #616070;
+  font-size: 12px;
+  margin: 0;
+}
```

Comment:
The class name 'price' is used for both a div and a paragraph, which can lead to confusion and potential styling issues. Consider using a more specific class name for the paragraph to differentiate it from the div.